release_util/management/commands/check_reserved_keywords.py

```diff
@@ -141,11 +161,14 @@ def __init__(self, reserved_keyword_config_file, override_file, report_path):
 
     @staticmethod
     def read_config_file(config_file_path):
-        click.echo("Loading config file: {}".format(config_file_path))
+        log.debug("Loading config file: {}".format(config_file_path))
```

Comment:
I think log.info is more parallel to click.echo.  I don't know about this logger, but loggers in general (software engineering) don't print debug statements by default.

Reply:
Right. These lines were helpful in development/testing, but I found them to be noisy and maybe not helpful in the CI flow.

Reply:
Gotcha, but I think the loading of the config file is useful and not noisy.

Reply:
fair enough. will change